pyatlan/generator/templates/methods/asset/atlas_glossary.jinja2

```diff
@@ -1,8 +1,18 @@
 
     @root_validator()
     def _set_qualified_name_fallback(cls, values):
```

Comment:
The code in these methods looks like it's duplicated. I would refactor it into a function.

Reply:
I have removed the code duplication from these methods, so now they look clean. I'll leave it as is. Thanks!